test/js/bun/shell/commands/cp.test.ts

```diff
@@ -0,0 +1,177 @@
+import { $ } from "bun";
+import { createTestBuilder } from "../test_builder";
+import { beforeAll, describe, test, expect, beforeEach } from "bun:test";
+import { sortedShellOutput } from "../util";
+import { tempDirWithFiles } from "harness";
+import fs from "fs";
+import { shellInternals } from "bun:internal-for-testing";
+const { builtinDisabled } = shellInternals;
+
+const TestBuilder = createTestBuilder(import.meta.path);
+
+const p = process.platform === "win32" ? (s: string) => s.replaceAll("/", "\\") : (s: string) => s;
+
+$.nothrow();
+
+describe.if(!builtinDisabled("cp"))("bunshell cp", async () => {
+  TestBuilder.command`cat ${import.meta.filename} > lmao.txt; cp -v lmao.txt lmao2.txt`
+    .stdout(p("$TEMP_DIR/lmao.txt -> $TEMP_DIR/lmao2.txt\n"))
+    .ensureTempDir()
+    .testMini()
+    .fileEquals("lmao2.txt", await $`cat ${import.meta.filename}`.text())
```

Comment:
If this throws, the rest of the tests in this file will not run and we will never know

Please move all of usages of top-level await in this test to be inside closures.